src/34_Command_Line_Interface.md

```diff
@@ -1,7 +1,7 @@
 # Command Line Interface
 
 Gitpod sports a command line interface that is available in each workspace terminal called `gp`:
-```
+```text
```

Comment:
What is the effect of adding this language specifier? Isn't `text` the default in mdBook?

Reply:
There was a build failure due to some unexpected lexical tokens when leaving it out.